walkthroughs/week-5-tdd/project/src/main/java/com/google/sps/FindMeetingQuery.java

```diff
@@ -15,9 +15,78 @@
 package com.google.sps;
 
 import java.util.Collection;
+import java.util.ArrayList;
```

Comment:
Make sure you are following the open source style guide

https://google.github.io/styleguide/javaguide.html